skiko/src/commonMain/kotlin/org/jetbrains/skiko/FrameDispatcher.kt

```diff
@@ -28,7 +28,9 @@ class FrameDispatcher(
 
     private val job = scope.launch {
         while (true) {
+            /// Await for draw request
```

Comment:
Why do you use `///` for comments?

Reply:
Apple uses /// or /* */ in SourceKit to generate and render documentation comments.

Reply:
All of them:
<img width="439" alt="Screenshot 2023-05-16 at 13 05 38" src="https://github.com/JetBrains/skiko/assets/4167681/d1ccafe0-d2f8-446d-a1cc-4331a417731d">

Reply:
The mentioned one is in Kotlin file though, I'll remove it.